Makefile.win

```diff
@@ -199,6 +208,49 @@ clean_crystal: ## Clean up crystal built files
 clean_cache: ## Clean up CRYSTAL_CACHE_DIR files
 	$(call RMDIR,"$(shell .\bin\crystal env CRYSTAL_CACHE_DIR)")
 
+.PHONY: rc
+rc: ## Write compiler resource script to standard output
+	$(eval comma := ,)
+	$(eval ver_comma := $(subst .,$(comma),$(subst -dev,,$(CRYSTAL_VERSION))))
+	$(eval build_date := $(shell git show -s --format=%cs HEAD))
+	$(eval build_year := $(firstword $(subst -, ,$(build_date))))

```

Comment:
I'd suggest to name these variables `source_date` and `source_year`. They do not represent the date of the build.
Also support for `SOURCE_DATE_EPOCH` would be nice (which is already used in this Makefile for the source date reported by the compiler itself).

Reply:
Conversion from a Unix timestamp is not possible with CMD nor with the built-in version of PowerShell (`Get-Date` on version 5.1 does not support the `-UnixTimeSeconds` parameter).

Reply:
One evil approach is:

```
$(eval source_date := $(shell crystal eval puts Time.unix($(SOURCE_DATE_EPOCH)).to_s(%(%Y-%m-%d))))
```

A more evil one is to run the following _Crystal_ code with the empty prelude inside the Makefile:

```crystal
lib LibC
  struct Tm
    tm_sec : Int32
    tm_min : Int32
    tm_hour : Int32
    tm_mday : Int32
    tm_mon : Int32
    tm_year : Int32
    tm_wday : Int32
    tm_yday : Int32
    tm_isdst : Int32
  end

  fun _gmtime64(timer : Int64*) : Tm*
  fun strftime(str : UInt8*, count : UInt64, format : UInt8*, time : Tm*) : UInt64
  fun printf(format : UInt8*, ...) : Int32
end

struct StaticArray(T, N)
  def to_unsafe
    pointerof(@buffer)
  end
end

class String
  def to_unsafe
    pointerof(@c)
  end
end

t = {{ env("SOURCE_DATE_EPOCH").to_i }}.to_i64!
tm = LibC._gmtime64(pointerof(t))
buf = uninitialized UInt8[11]
LibC.strftime(buf, 11, "%Y-%m-%d", tm)
LibC.printf("%s", buf)
```